src/screens/Profile/Profile.tsx

```diff
@@ -2,10 +2,17 @@ import { Box, Typography } from '@mui/material';
 import { Navigate, useParams } from 'react-router-dom';
 
 import { useUserByUsername } from '../../Api/EndPoints/useUserByUsername';
+import DEFAULT_AVATAR from '../../assets/defaultImage.jpg';
 import { UserProfileDescription } from '../../components';
 import { ROUTES } from '../../routes/Routes';
 import { CDN } from '../../utils/CDN';
 
+const isAvatarValid = (avatarUrl: string | undefined | null) => {
+    return (
+        avatarUrl !== null && avatarUrl !== undefined && avatarUrl.trim() !== ''
```

Comment:
if you just

If(avatarUrl) ... => that would check if it's empty string, null, undefined, it's checking for nullish value

That whole function is overkill and redundant

Reply:
ok, change it